librz/core/casm.c

```diff
@@ -120,17 +121,24 @@ RZ_API RzCmdStatus rz_core_asm_plugin_print(RzCore *core, RzAsmPlugin *ap, const
 			strcat(bits, "64");
 		}
 	}
-	feat = "__";
+
 	if (ap->assemble && ap->disassemble) {
-		feat = "ad";
+		strcpy(feat, "ad");
 	}
 	if (ap->assemble && !ap->disassemble) {
-		feat = "a_";
+		strcpy(feat, "a_");
 	}
 	if (!ap->assemble && ap->disassemble) {
-		feat = "_d";
+		strcpy(feat, "_d");
 	}
 	feat2 = has_esil(core, ap->name);
+	strcat(feat, feat2);
+
+	for (int i = 0;flags && flags[i] != '\0'; i++) {
```

Comment:
I like how you solved it, but this is not clang formatted. Run sys/clang-format.py (requires clang format) and submit the changes.